src/Agent.Plugins/PipelineArtifact/PipelineArtifactProvider.cs

```diff
@@ -10,11 +10,23 @@
 using Agent.Sdk;
 using System.Threading;
 using System.Linq;
+using Microsoft.TeamFoundation.DistributedTask.WebApi;
 
 namespace Agent.Plugins.PipelineArtifact
 {
     internal class PipelineArtifactProvider : IArtifactProvider
     {
+        private const int DefaultDedupStoreClientMaxParallelism = 192;
```

Comment:
Just curios why we set MaxParallelism to 192

Reply:
Ah NVM just saw your comment on the top.

Reply:
I should add a note to the code :)